lib/drand.ts

```diff
@@ -0,0 +1,69 @@
+import HTTP, {ClientOptions, Watcher} from './http'
+import VerifyingClient from './verifying-client'
+import OptimizingClient from './optimizing-client'
+
+export type DrandOptions = Partial<{
+    chainHash: string
+    insecure: boolean
+    disableBeaconVerification: boolean
+    chainInfo: ChainInfo
+    cacheSize: number
+    watcher: Watcher
+}>
+
+export interface NetworkClient {
```

Comment:
Missing a `chains` function to get the list of chains supported by the node.

Reply:
have raised https://github.com/drand/drand-client/issues/34 to address this - right now I want to limit adding new functionality in this PR